CHANGELOG.md

```diff
@@ -25,58 +25,29 @@
 
 All notable changes to this project will be documented in this file.
 
+## 1.3.3 - 2021-02-26
 ## Unreleased
```

Comment:
I think this line needs to be deleted from here, right?

Reply:
Good catch! fixed.